main.py

```diff
@@ -98,7 +99,6 @@
 get_profile_traffic = Template("""/repos/$owner/$repo/traffic/popular/referrers""")
 truthy = ['true', '1', 't', 'y', 'yes']
 
-

```

Comment:
I don't think this change is vital. Also it breaks [pep](https://peps.python.org/pep-0008/#blank-lines).